python/doc/content/python/peacock.md

```diff
@@ -1,33 +1,86 @@
 # Peacock
 
+!style! halign=left
 Peacock is a graphical front end for the MOOSE input file syntax. Peacock allows the user to build or modify an input file, execute the application and view the results all within one package.
+!style-end!
+
 
 ## Conda Packages
 
 !include installation/install_peacock.md
 
 ## Environment
 
-Adding Peacock to yout PATH will allow you to launch it from your application or "tests" directory. Peacock will search up through the filesystem directory tree until it finds your application, obviating the need th specify your application as an argument, and simplifying Peacock usage.
+!style! halign=left
+Adding Peacock to yout PATH will allow you to launch it from your application or `tests`` directory. Peacock will search up through the filesystem directory tree until it finds your application, obviating the need to specify your application as an argument, and simplifying Peacock usage.
+!style-end!
+
+First, we need to discover the absolute path to `peacock` on your machine. Assuming that your MOOSE
+repository sits in the default location of `~/projects/moose`, enter the following directory and run
+`pwd`. Example:
+
+```bash
+cd ~/projects/moose/python/peacock
+pwd
+/home/you/projects/moose/python/peacock
+```
+
+Whatever returns, this is your path to `peacock`.
 
-To add Peacock yo your PATH: assuming that $ MOOSE_DIR is the location of your git clone, navigate to the `$MOOSE_DIR/python/peacock` directory, and type: pwd. Whatever is printed out is the path to Peacock. Next, add the following to the end of your ~/bash_profile (Macintosh) or ~/.bashrc (Ubuntu) file:
+Next, we need to know what shell you are operating in to determine what profile to modify.
+Run `echo $0` to do so.
+Example:
 
 ```bash
-export PATH=pwd:$PATH
+echo $0
+-zsh
 ```
 
-Where "pwd" is what was printed out in the previous step. Note that there can be *no spaces* around the equals sign in the line above!
+Where `-zsh` is our example result. The following table describes which profile file one must edit,
+as well as how to assign a value to a variable in that shell:
+
+| Shell | Profile Location | Command |
+| :- | :- | :- |
+| bash | `~/.bash_profile` | `export PATH=pwd:$PATH` |
+| zsh | `~/.zshrc` | `export PATH=pwd:$PATH` |
+| sh | `~/.profile` | `export PATH=pwd:$PATH` |
+| csh | `~/.cshrc` | `setenv PATH pwd:$PATH` |
+| tcsh | `~/.tcshrc` | `setenv PATH pwd:$PATH` |
+
+Replace occurrences of `pwd` in the above table with the result you recieved in the previous step,
+as you apply the following instructions:
+
+!alert! warning title=Dangerous Syntax Below
+Use single quotes: ( `'` ). We *literally* want to write "`$PATH`" in the below example, and not the
+value of `$PATH`.
+
+Use two greater thans: ( `>>` ). Two greater thans append information to a file, while a single
+greater than ( `>` ) overwrites the file entirely.
+!alert-end!
```

Comment:
We might include a comment here about backing up the file before doing this if we're highlighting how dangerous this is.

Reply:
The newly-added comment is sufficient, I think. Thanks!